src/components/pages/InstructorHome/CourseCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Card, Avatar, Badge } from 'antd';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+const { Meta } = Card;
+
+const CourseCard = props => {
+  return (
+    <>
+      <Link to="/classroom">
+        {/* <Card className="course__card" hoverable>
```

Comment:
There are a few esLint warnings. Should delete them if you're not using it. Like 'connect', 'Meta', 'Badge', 'Avatar'.